docker/Dockerfile

```diff
@@ -53,6 +53,7 @@ COPY . /orderly
 RUN R CMD INSTALL /orderly && \
   Rscript -e 'orderly:::write_script("/usr/bin")' && \
   cp /orderly/inst/create_orderly_demo.sh /usr/bin/ && \
+  cp /orderly/inst/run_orderly_demo /usr/bin/ && \
```

Comment:
I've left the old script `create_orderly_demo.sh` with the extension, since it could be in use by other repos